ai/fine-tuning-v1/and_query.shs

```diff
@@ -0,0 +1,26 @@
+{
+  prompt: """
+    How do I check more than one condition?
+  """
+
+  solution: {
+    variable
+    When({Is(condition1) | And | variable2 | Is(condition2)} {
+      ;; DoSomething
```

Comment:
Better to use `Pass` cos it will fail to parse without a shard here! so more like
```
Pass ; Do Something
```